cmd/tools_veth.go

```diff
@@ -13,6 +13,7 @@ import (
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/srl-labs/containerlab/clab"
+	"github.com/srl-labs/containerlab/links"
```

Comment:
looks like all this should leverage proper endpoint types and not the ones from types package?

Reply:
As discussed, seperate PR